### PR TITLE
docs: updated README.md according to best practices

### PR DESCRIPTION
**Pull request by pgrzy-golem**
JST-393
JST-394

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,201 @@
...
+| URL to arbitrary download location | hash is needed               | `https://example.com/my-image`                                                                |                                                                                                   |
+
+If hash is not provided or resolved, you will get a warning that the manifest will not be usable until you provide it manually.
+
+
+### Adding outbound URLs
+
+In order to be able to access the internet from Golem network, your application needs to declare the outbound URLs it will be using inside its manifest.
+
+There is a default set of URLs that providers may allow your application to use (TODO: link to default URL whitelist).
```
Comment (cryptobench) [inline README.md:103]: Whitelisted domains/ips can be found here

https://github.com/golemfactory/ya-installer-resources/tree/main/whitelist

File: README.md [hunk 2]
```diff
@@ -1,27 +1,201 @@
...
+In order to use URLs from outside of this whitelist, you need to provide a signed manifest that can be validated by certificates issued by Golem.
+
+**NOTE:** Currently there is no process for obtaining the certificate needed to validate the manifest signature. Please contact us on discord if you need to use URLs outside the default whitelist.
```
Comment (cryptobench) [inline README.md:106]: Write discord like this to link it.

```markdown
[Discord](https://chat.golem.network)
```

File: README.md [hunk 3]
```diff
@@ -1,27 +1,201 @@
...
+
+```shell
+golem-sdk manifest net add-outbound https://golem.network https://github.com https://example.com
+```
+
+### Signing the manifest
+
+In order to use URLs outside the default whitelist, you need to sign the manifest with a key provided by Golem.
+
+**NOTE:** Currently there is no process for obtaining the certificate needed to validate the manifest signature. Please contact us on discord if you need to use URLs outside the default whitelist.
```
Comment (cryptobench) [inline README.md:132]: Write discord like this to link it.

```markdown
[Discord](https://chat.golem.network)
```

File: README.md [hunk 4]
```diff
@@ -1,27 +1,201 @@
...
+The Pull Request should describe what changes you've made, what's the purpose of them and how to use them.
+
+
+## Links
+
+- Project homepage: https://github.com/golemfactory/golem-sdk-cli
+- Repository: https://github.com/golemfactory/golem-sdk-cli
+- Issue tracker: https://github.com/golemfactory/golem-sdk-cli/issues
+    - In case of sensitive bugs like security vulnerabilities, please contact
+      my@email.com directly instead of using issue tracker. We value your effort
```
Comment (cryptobench) [inline README.md:179]: myemail needs to be replaced. Did you figure out anything with Staziek?

File: README.md [hunk 5]
```diff
@@ -1,27 +1,201 @@
...
+      to improve the security and privacy of this project!
+- [Golem](https://golem.network), a global, open-source, decentralized supercomputer that anyone can access.
+- [Golem Image Registry](https://registry.golem.network)
+- [Golem Discord](https://discord.gg/golem)
+- Documentation:
+    - [Quick start](https://docs.golem.network/creators/javascript/quickstart/) for JavaScript developers
```
Comment (cryptobench) [inline README.md:185]: fix link

https://docs.golem.network/docs/creators/javascript/quickstarts

File: README.md (hunk 4 above)
Comment (pgrzy-golem) [reply README.md:179]: he didn't reply

**Comment by jalas167** [comment]
I have added some changes directly as new commits.

**Review by jalas167 — COMMENTED** [review]
Please change the licence to GPL

**Review by cryptobench — APPROVED** [review]

**Pull request merged**